service/awsconfig/v5/resource_set.go

```diff
@@ -248,6 +265,7 @@ func NewResourceSet(config ResourceSetConfig) (*framework.ResourceSet, error) {
 		kmsKeyResource,
 		s3BucketResource,
 		s3BucketObjectResource,
+		loadBalancerResource,
```

Comment:
Resource runs before the cloudformation resource to do the GC. Otherwise the ELBs will be in the public subnet and the CloudFormation stacks will not delete cleanly.